lib/ansible/modules/monitoring/standup.py

```diff
@@ -0,0 +1,533 @@
+#!/usr/bin/python
+
+# Copyright (c) 2017, Thomas K. Theakanath <thomastk@gmail.com>
+
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['stableinterface'],
```

Comment:
This should be 'preview' status probably?

Reply:
The interface to the module is exepected to change as this module is an Ansible version of an infra verification tool that I had developed earlier. The module is developed from scratch and yet to be used extensively in the field. Would that make the status of the module "preview"?